### PR TITLE
openstack-tests: Add branches

### PR DESCRIPTION
**Pull request by pierreprinetti**
Add branches for 4.10 through 4.5.
Add placeholder images for older releases.

Implements OSASINFRA-2682

### CODE REVIEW

File: ci-operator/config/openshift/openstack-test/openshift-openstack-test-release-4.1.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  cat > /bin/openstack-tests <<EOF
+  #!/usr/bin/env bash
+  echo 'Not implemented'
+  EOF
+  chmod +x /bin/openstack-tests
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
```
Comment (mandre) [inline ci-operator/config/openshift/openstack-test/openshift-openstack-test-release-4.1.yaml:11]: Please check the golang version each branch is pinned to.

Comment (pierreprinetti) [reply ci-operator/config/openshift/openstack-test/openshift-openstack-test-release-4.1.yaml:11]: I think 1.16 is fine for all branches. Do you think we should change it?

Comment (mandre) [reply ci-operator/config/openshift/openstack-test/openshift-openstack-test-release-4.1.yaml:11]: Not sure, I was under the impression each ocp release was capped to a golang version but I might be mistaken.

**Review by mandre — COMMENTED** [review]
/lgtm

**Pull request merged**